modules/core/src/main/java/org/apache/ignite/ssl/SslContextFactory.java

```diff
@@ -476,6 +477,7 @@ private void checkNullParameter(Object param, String name) throws SSLException {
      * @return Opened input stream.
      * @throws IOException If stream could not be opened.
      */
```

Comment:
```suggestion
     * @deprecated Use {@link = U#openFileInputStream(String)} instead.
     */
```

Reply:
we can't offer to use internal class from public API